src/vs/workbench/contrib/browserView/electron-browser/media/browser.css

```diff
@@ -0,0 +1,113 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I would suggest to use CSS nesting here to ensure the selectors are encapsulated properly. Currently they seem very flat and might potentially match something else?